workloads/logging/deploy_logging_stack.sh

```diff
@@ -37,22 +37,15 @@ function install() {
   fi
 }
 
-wait_time=0
 function cleanup() {
-  oc delete --wait=true project openshift-logging --ignore-not-found
-  oc delete --wait=true project openshift-operators-redhat --ignore-not-found
-  while [[ $( oc get projects | grep -w "openshift-logging\|openshift-operators-redhat") ]]; do
-    sleep 5
-    wait_time=$((wait_time+5))
-    if [[ $wait_time -ge $TIMEOUT ]]; then
-      log "openshift-logging/openshift-operators-redhat namespaces still exists after $TIMEOUT, please check. Exiting"
-      exit 1
-    fi
-  done
+  if ! oc delete --wait=true project openshift-logging openshift-operators-redhat --ignore-not-found --timeout=180s; then
```

Comment:
Why don't you use the variable ${TIMEOUT} rather than hardcoded 180s?

Reply:
because we were using it wrong. TIMEOUT is the time of log_generator to wait for all messages to be stored, it is being used `[here](https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/logging/files/log_generator.yaml#L20)`. We set it to HOURS, we cannot use it for deleting objects.

Also, in this same file, we have another wait hardcoded to 180, `[here](https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/logging/deploy_logging_stack.sh#L68)`

Reply:
ok!, can you update the docs then?, According to the docs
```
### TIMEOUT
Default: 180

Time to wait for resources created/deleted before exiting
```

Reply:
done